Documentation/networking/overview.md

```diff
@@ -332,7 +332,14 @@ The pod's TCP port 80 can be mapped to an arbitrary port on the host during rkt
 # rkt run --port=http:8888 myapp.aci
 ```
 
-Now, any traffic arriving on host's TCP port 8888 will be forwarded to the pod on port 80.
+Now, any traffic arriving on host's TCP port 8888 on any IP will be forwarded to the pod on port 80.
+
+
+You can also forward a port that is not referenced at the app time with the `--raw-port` option.  The syntax is Name, Protocol (`tcp` or `udp`), Pod port, Host IP (use 0.0.0.0 for all IPs), Host port.
```

Comment:
What is "Name" here and what is the user supposed to pick for it? Can we do better and just generate one for the user? Or ask the user to name them `port-proto` so we can automatically pick them up without a new option?